src/main/features/offer/index.ts

```diff
@@ -0,0 +1,28 @@
+import * as express from 'express'
+import * as path from 'path'
+
+import { RouterFinder } from 'common/router/routerFinder'
+import { ClaimMiddleware } from 'app/claims/claimMiddleware'
+import { AuthorizationMiddleware } from 'idam/authorizationMiddleware'
+import { AuthenticationRedirectFactory } from 'utils/AuthenticationRedirectFactory'
+
+function requestHandler (): express.RequestHandler {
+  function accessDeniedCallback (req: express.Request, res: express.Response): void {
+    res.redirect(AuthenticationRedirectFactory.get().forLogin(req, res))
+  }
+
+  const requiredRoles = [
+    'cmc-private-beta',
+    'defendant'
```

Comment:
I don't think this should be here? these pages should be accessible to anyone in the private beta I think?

Reply:
as per the story it's only accessible to defendant from response confirmation page.

Reply:
and on the defendant dashboard page? which is missing from this.
But this 'offer' feature is to be used by both claimant's and defendants..

Reply:
not as per the story, unless we change it.

Reply:
![image](https://user-images.githubusercontent.com/21194782/31932966-b3371bfe-b89f-11e7-9cfc-cc79fe975bd5.png)

Reply:
seems to be there to be and I can't see anything about only a defendant is able to access the offer feature..

Reply:
defendant's dashboard, that says it all.

Reply:
Claimant will have an option to make a counter offer. We can either make these pages available for claimant now or when the story to make counter-offer is played, both are fine for me.

Reply:
the main part is we shouldn't be making decisions based on roles like these, we should be checking whether the claimant is allowed to access the specific offer not that they are allowed to use the offer feature.

Reply:
please remove, it is unnecessary for this check to be here.